### ISSUE TITLE
Migrate to Sanity Create Template

### ISSUE THREAD

**Issue by jamie-endeavour** [HeySugar/hey-sugar-app#3]
I've been speaking to the folks at Sanity and have been told I can publish a template for HeySugar through https://sanity.io/create. This is awesome because it means the getting started process will be much better/easier/awesome.

**Comment by jamie-endeavour** [comment]
I just wanted to leave a little update. I started working on this yesterday evening and I've successfully been able to use the Sanity Template development tool to create a build folder which contains pre-populated information.

My next task is to reach out to Sanity next week and see about testing the deployment and tying up the loose ends. In the mean time I'll improve the documentation and get get this repository updated to contain the new template.

Thank you to those who have starred the project so far, not long now before we can see this thing in full swing!
